x-pack/legacy/plugins/advanced_ui_actions/public/np_ready/kibana.json

```diff
@@ -2,7 +2,8 @@
   "id": "advanced_ui_actions",
   "version": "kibana",
   "requiredPlugins": [
-    "embeddable"
+    "embeddable",
+    "ui_actions"
```

Comment:
I think this will need to be `uiActions`, but until it's used... not really important.